primer.md

```diff
@@ -128,6 +129,34 @@ The following will not be part of the specification:
 * Language-specific runtime APIs
 * Selecting a single identity/access control system
 
+
+## Architecture
+
+The CloudEvents specification set defines four different kinds of protocol
+elements that form a layered architecture model.
+
+1. The [base specification](spec.md) defines an abstract information model and
+   associated rules for what constitutes CloudEvent.
```

Comment:
It might be good if `1.` introduces "attributes", otherwise they are a bit abruptly introduced in `2.`.